app/controllers/api/experimental/concerns/column_data.rb

```diff
@@ -180,6 +180,7 @@ def sum_columns(column_names, work_packages)
     end
   end
 
+  # FIXME: Deprecated. Should be removable once the experimental WP controller is gone
```

Comment:
Oh, that is the sad part of this PR... It will only add stuff, but not remove the experimental API, because it is still being used...